logforwarder/src/apps/vespa-logforwarder-start/cf-handler.cpp

```diff
@@ -76,6 +77,17 @@ CfHandler::doConfigure()
     fclose(fp);
     rename(tmpPath.c_str(), path.c_str());
 
+    path = cfFilePath(config.splunkHome, "inputs.conf");
+    tmpPath = path + ".new";
+    *fp = fopen(tmpPath.c_str(), "w");
+    if (fp == NULL) return;
+
+    fprintf(fp, "[default]\n");
+    fprintf(fp, "host = %s\n", getenv("HOSTNAME"));
+    fprintf(fp, "_meta = vespa_tenant::%s vespa_application::%s vespa_instance::%s\n", getenv("VESPA_TENANT"), getenv("VESPA_APPLICTAION"), getenv("VESPA_INSTANCE"));
```

Comment:
must be VESPA_APPLICATION here.
also, we should probably detect if any of these are unset (getenv returns null pointer), i guess...